doc/user/content/get-started.md

```diff
@@ -11,10 +11,8 @@ To help you get started with Materialize, we'll:
 - Explore its API
 - Set up a real-time stream to perform aggregations on Wikipedia user edits.
 
-{{< cta >}}
-
-<a href="https://www.katacoda.com/materialize/scenarios/intro-wikipedia" target="_blank">Click here to try the demo in your browser!</a>
-
+{{< cta href=/katacoda?intro-wikipedia target=_blank >}}
+Click here to try the demo in your browser!
```

Comment:
I strongly prefer "Try the demo in your browser →"

I, too, have an aversion to "Click here" in the head of a link. I also have an aversion to exclamation points in copy.

Reply:
It is done.